jekyll/_cci2/server-3-install-migration.adoc

```diff
@@ -10,10 +10,10 @@ version:
 :toc: macro
 :toc-title:
 
-Migrating from 2.19.x to 3.0 requires that you backup your 2.19 installation and then restore the data to the new Server 3.0 installation.
-It does require an already operating Server 3.0 installation. Depending on the size of your data stores, the migration
-can take anywhere from a few minutes to a few hours. We recommend using a staging environment before completing this
-process in a production environment. This will not only allow you to gain a better understanding of the migration process,
+Migrating from 2.19.x to 3.0 requires backing up your 2.19 instance data (Mongo, Postgres, and Vault) and than restoring that data in a waiting Server 3.0
+instance. If you run into trouble you are able to fallback to your 2.19 instance. It does require an already operating Server 3.0 installation. Depending 
```

Comment:
I'm not clear on what "It does require an already operating Server 3.0 installation" means here - the fallback process doesn't require an operational 3.0 installation?

Reply:
Oh I see, that just means an operational 3.0 install is required before starting the migration.